.go-version

```diff
@@ -1 +1 @@
-1.21.7
+1.22.0
```

Comment:
1.22.1 is [already released](https://groups.google.com/g/golang-announce/c/5pwGVUPoMbg)

```suggestion
1.22.1
```